samples/Emacs Lisp/filenames/Cask

```diff
@@ -0,0 +1,8 @@
+(package "cask-example" "1.2.3" "Example package for Emacs Cask")
```

Comment:
@pchaigno I can provide [my project Cask file](https://github.com/zonuexe/composer.el/blob/master/Cask).
(I avoided to use [original project Cask file](https://github.com/cask/cask/blob/master/Cask) that is licensed under GPLv3.)